packages/offix-client/src/config/ApolloOfflineClientConfig-ti.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * This module was automatically generated by `ts-interface-builder`
+ */
+import * as t from "ts-interface-checker";
+// tslint:disable:object-literal-key-quotes
+
+export const ApolloOfflineClientConfigIntf = t.iface(["ApolloOfflineClientOptions"], {
```

Comment:
I tried this using the react example app, and it complains about `ApolloOfflineClientOptions` being unknown. So it seems that the interface checker might not understand anything beyond basic types. This means we might need an interface checker for all of the classes in this file.
i.e. we'd have to create an `ApolloOfflineClientOptions-ti` file, the same for an `ApolloOfflineQueueListener-ti` file and so on.

cc @darahayes and @wtrocki what are your thoughts on going down the rabbit hole of creating an interface checker for each of these classes?